test/data/metapackages/NonConformingName/CHANGELOG.rst

```diff
@@ -0,0 +1,163 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Neither of these added changelog files seems to be related to `flake8` checking?

Reply:
Not related at all.  Sorry.  Reverted.